From 54d447640d3c550143e07610a516e4cdf474260e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Daniel=20P=2E=20Berrang=C3=A9?= <berrange@redhat.com>
Date: Wed, 3 Jul 2024 13:47:04 +0100
Subject: Revert "meson: temporarily disable -Wunused-function"
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This reverts commit c682111eaa73d9b985187b8be330338f50b78a7a.

No longer needed after introduction of downstream machines.

Signed-off-by: Daniel P. Berrangé <berrange@redhat.com>
---
 meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/meson.build b/meson.build
index 38ad60fc10..0607c1313b 100644
--- a/meson.build
+++ b/meson.build
@@ -747,7 +747,6 @@ warn_flags = [
   '-Wno-string-plus-int',
   '-Wno-tautological-type-limit-compare',
   '-Wno-typedef-redefinition',
-  '-Wno-unused-function',
 ]
 
 if host_os != 'darwin'
-- 
2.39.3

